docs/source/examples/relaxation.py

```diff
@@ -0,0 +1,38 @@
+"Relaxation examples"
+
+from gpkit import Variable, Model
+x = Variable("x")
+x_min = Variable("x_min", 2)
+x_max = Variable("x_max", 1)
+m = Model(x, [x <= x_max, x >= x_min])
+print "Original model"
+print "=============="
+print m
+print
+# m.solve()  # raises a RuntimeWarning!
+
+print "With constraints relaxed equally"
+print "================================"
+from gpkit.constraints.relax import RelaxAll
+relaxall = RelaxAll(m)
```

Comment:
I'm not convinced that RelaxAll will be appropriate to use for any real Model. There are going to be some constraints (e.g. equalities) that are simply not appropriate to relax.

I'm advocating this because I'm thinking it might make more sense for the relaxation to happen at the ConstraintSet level, instead of the Model level. And I think this might help with the circular imports.